app/views/kaminari/_first_page.html.erb

```diff
@@ -0,0 +1,11 @@
+<%# Link to the "First" page
```

Comment:
Are the materialized views necessary?

Reply:
Do you mean, "is it necessary to have run the kaminari view generator and have these files in the codebase" vs "Just using the pagination helpers and letting kaminari do its thing?"

If so the answer is yes and no. Yes we want to run this generator and override these views with ones that follow bootstraps conventions to get the nice "bootstrapy" looking pagination. Otherwise we just get raw `a` tags that dont have bootstraps styles applied.

There is an alternative in installing an additional gem that does this for us, but it seemed simple enough and felt worth it to avoid and additional dependency.

Reply:
AHHHHH

All the views looked completely normal and sensible; I hadn't considered they might be less good looking by default.